daily_read/daily_report.py

```diff
@@ -23,11 +23,12 @@
 }
 
 STATUS_DESCRIPTIONS = {
-    "All Raw data Delivered": "The data has been made available through NGIs delivery system",
-    "All Samples Sequenced": "Sequencing (including potential resequencing) of all samples has been finished",
+    "All Raw data Delivered": "The data has been made available through NGIs delivery system.",
+    "All Samples Sequenced": "Sequencing (including potential resequencing) of all samples has been finished.",
     "Library QC finished": "Library QC is a quality control of the sequencing library produced either by NGI or supplied by you, depending on the type of project.",
     "Reception Control finished": "Reception Control consists of NGI staff measuring e.g. concentration and volume for the samples received.",
-    "Samples Received": "The samples have been received and registered at NGI",
+    "Samples Received": "The samples have been received and registered at NGI.",
+    "Pending": "The order has been set up but the samples have not yet been received by NGI.",
```

Comment:
```suggestion
    "Pending": "The order has been set up but the samples have not yet been received or registered by NGI.",
```

Reply:
Not sure if I should use "and" or "or" in my suggestion though. You probably know better.

Reply:
I guess 'or' is better? They won't show up until we enter it in LIMS.